app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -172,25 +172,18 @@ class PatientRepositoryAndroidTest {
 
```

Comment:
Spelling typo in the commit message "seach"

Reply:
Fixed.